alfred-telemetry-solr/alfred-telemetry-solr-common/src/main/java/eu/xenit/alfred/telemetry/solr/monitoring/binder/SolrTrackerMetrics.java

```diff
@@ -18,7 +18,7 @@
     AlfrescoCoreAdminHandler coreAdminHandler;
     MeterRegistry registry;
 
-    Logger logger = LoggerFactory.getLogger(SolrMetrics.class);
+    Logger logger = LoggerFactory.getLogger(SolrTrackerMetrics.class);
```

Comment:
Logger should be at least `private`, preferably also `static final`

* `private` - so that no other class can hijack your logger
* `static` - so there is only one logger instance per class, also avoiding attempts to serialize loggers
* `final` - no need to change the logger over the lifetime of the class

https://stackoverflow.com/a/6653577